includes/application.php

```diff
@@ -33,6 +33,24 @@
     $app->bind(HandlerInterface::class, 'error.handler.development');
     ini_set('display_errors', true);
     error_reporting(E_ALL);
+
+    // setup var dump server to use for easier debugging
+    $vardumpServerConfig = config('vardump_server');
+    if ($vardumpServerConfig['enable'] === true) {
+
+        $cloner = new Symfony\Component\VarDumper\Cloner\VarCloner();
+        $fallbackDumper = \in_array(\PHP_SAPI, ['cli', 'phpdbg']) ? new Symfony\Component\VarDumper\Dumper\CliDumper() : new Symfony\Component\VarDumper\Dumper\HtmlDumper();
```

Comment:
You couldnt you just use `\Symfony\Component\VarDumper\VarDumper` here?

And using `imports` shortens line lengths :grin:)